client/src/charts/canada/canada_d3_component.js

```diff
@@ -36,7 +36,7 @@ const ordering = {
   pe: 16,
 };
 
-export class Canada {
+export class Canada_D3_Component {
```

Comment:
Little nitpick (this is my fault, I suggested the wrong name for this file in me earlier feedback), by convention this (and the file name) would be CanadaD3Component. I forgot it was a class.

Reply:
oh true yup yup